content/blog/2014/sqlalchemy-postgres-autocommit.md

```diff
@@ -20,6 +20,21 @@ summary: |
 date: 2014-06-14
 ---
 
+{% import 'utility.macros.njk' as utility %}
+
+{% set update = ['Update', utility.datetime('2024-04-26')] | join(' ') %}
+{% callout 'note', update %}
```

Comment:
We should file a new issue to simplify this in the future, but for now looks great.